app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java

```diff
@@ -5,176 +5,128 @@
 /**
  * Created by juan.herrera@globant.com on 11/08/2016.
  */
-public class SubscribersModel {
+public final class SubscribersModel {
 
-    @SerializedName("login")
-    private String login;
-    @SerializedName("id")
-    private String id;
+    final String login;
```

Comment:
Private access modifier must be here, this is, `private final String login`. Do the same for the rest of the fields/attributes.